.mvn/settings-cucumber-jvm.xml

```diff
@@ -0,0 +1,6 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <localRepository>${user.home}/.m2/repository-cucumber-jvm</localRepository>
```

Comment:
> Configure an isolated local Maven repository cache using the maven -s option in .mvn/maven.config (Since maven 3.3.1+) - see https://maven.apache.org/configure.html

I'm missing a motivating reason. Unless you are doing terrible things with your cache this should not be needed.

Please remove.

Reply:
Maybe this is the wrong place to discuss what is terrible and what is not :) However - not terrible is having total clarity about what project the artifacts belong to. I have more than one of them and space problems are no longer an argument.

Reply:
I'm not convinced.

The current size of my personal Maven cache is 5GB. Professionally around 15GB. These are not trivial amounts of data. Especially not once we add duplicates for each project. People joke about `node_modules` and for good reason.

Not to mention the practical aspects. 

- Using individual caches will prevent the use of snapshot versions build locally. This means that snapshot builds can not be used in another project. For the Cucumber org this could be solved by adding the same setup to all other java projects, but then we are losing the advantage of using Maven defaults by having to configure all these projects. And it still doesn't cover the use of snapshots in non-Cucumber Org projects.

- Using individual caches prevents the reuse of common dependencies already downloaded. Given that a significant part of Cucumber was developed on poor internet connections I've come to appreciate this efficiency.

Please remove this the MR.